src/harmony/HistoryRecord/HistoryRecord.module.css

```diff
@@ -97,4 +94,14 @@
 
 .HistoryRecordCollapse + .HistoryRecord {
     align-items: center;
-}
\ No newline at end of file
+}
+
+.HistoryRecordCollapse {
+    height: 0;
+    overflow: hidden;
+    transition: height var(--transition-func) var(--transition-time);
```

Comment:
I thought `transition: height` to `auto` doesn't work in css

Reply:
yeah
height in css file are redundant styles
i removed it now